CactbotOverlay/FFXIVProcess.cs

```diff
@@ -118,8 +118,8 @@ public enum EntityJob {
       SAM = 34,
       RDM = 35,
       BLU = 36,
-      DNC = 37,
-      GNB = 38,
+      GNB = 37,
```

Comment:
Can you remove this from this pull request? I'm pretty sure this is correct, but I want to double check this myself before committing more plugin changes.

Reply:
should be reverted now